.eslintrc

```
@@ -0,0 +1,58 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "airbnb",
+    "plugin:@typescript-eslint/recommended",
+    "prettier" // prettier *has* to be the last one, to avoid conflicting rules
+  ],
+  "ignorePatterns": ["pnpm-lock.yaml"],
+  "plugins": ["simple-import-sort", "@typescript-eslint"],
+  "rules": {
+    "quotes": ["error", "double"],
+    "react/react-in-jsx-scope": "off", // next does not require react imports
+    "import/extensions": "off", // file extension not required when importing
+    "react/jsx-filename-extension": "off",
+    "no-restricted-syntax": "off",
```

Small suggestion to disallow `for ... in` which could be problematic when iterating over objects - for ... in iterates over keys of an object, while for ... of iterates over values of an iterable object which. for ... in behavior can always be achieved with `Object.keys()` which IMHO is more clear

```suggestion
    "no-restricted-syntax": [
      "error",
      {
        selector: "ForInStatement",
        message: "for ... in disallowed, use for ... of instead",
      },
    ],

```